internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -137,6 +158,12 @@ func resourceKustoCluster() *pluginsdk.Resource {
 				},
 			},
 
+			"restrict_outbound_network_access": {
```

Comment:
we usually put the verb on the end pf booleans
```suggestion
			"outbound_network_access_restricted": {
```